### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+project(RINGMeshPluginExample)
+
+# Turn on the ability to create folders to organize projects and files
+# It creates "CMakePredefinedTargets" folder by default and adds CMake
+# defined projects like INSTALL.vcproj and ZERO_CHECK.vcproj
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+# Define version number
+# It is then exported to the configuration file
+set(RINGMeshPluginExample_VERSION_MAJOR 5)
```
Comment (panquez) [inline CMakeLists.txt]: Does the version of the plugin correspond to the RINGMesh version used as base?

File: cmake/RINGMeshPluginExampleConfig.cmake.in [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+find_package(Geogram REQUIRED)
+
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_basic_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_mesh_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_geogram_extension_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_geomodel_core_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_geomodel_builder_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_tetrahedralize_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_geomodel_tools_target.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/RINGMesh_io_target.cmake)
```
Comment (panquez) [inline cmake/RINGMeshPluginExampleConfig.cmake.in]: Are these includes useful?

**Comment by panquez** [comment]
Do you think we need to add a LICENCE file for the plugin exemple?

**Comment by BotellaA** [comment]
At first, I did add a LIXENSE file but I assumed we want something in the header of each file...

**Review by panquez — APPROVED** [review]

**Pull request merged**